src/astx/base.py

```diff
@@ -113,6 +113,7 @@ class ASTKind(Enum):
     FunctionAsyncDefKind = -405
     AwaitExprKind = -406
     YieldExprKind = -510
+    YieldFromExprKind = -1207
```

Comment:
```suggestion
    YieldFromExprKind = -407
```

In theory we have a negative sequence grouped, which group has a range of 100 .. this is just to avoid repetitions .. it is not actually really important.